OWML.Common/Menus/IModPromptButton.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OWML.Common.Menus
+{
+    public interface IModPromptButton : IModButton
+    {
+        string DefaultTitle { get; }
+        ScreenPrompt Prompt { get; set; }
+
+        new IModPromptButton Copy();
+        IModPromptButton Copy(string title);
+        IModPromptButton Copy(string title, int index);
+
+        IModPromptButton Duplicate(string title);
+        IModPromptButton Duplicate(string title, int index);
+
+        IModPromptButton Replace(string title);
+        IModPromptButton Replace(string title, int index);
```

Comment:
I suggest leaving this out, until/unless you need them. In many cases I don't think we end up using these, and many UI elements don't have/need them. We have the base stuff anyways

Reply:
yeah, makes sense

Reply:
since it's basically only intended for sugarcoating existing buttons

Reply:
and if someone wants to copy it (s)he would probably also want to set new Prompt